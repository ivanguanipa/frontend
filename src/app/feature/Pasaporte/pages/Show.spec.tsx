import * as React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';
import ShowPage  from './Show';
import { Route, Switch } from 'react-router-dom';

describe('ShowPage Test', () => {
  let componentWrapper: ShallowWrapper;

  afterEach(() => {
    componentWrapper.unmount();
  });

  it('should match snapshot', () => {
    componentWrapper = shallow(
        <Route path="/pasaportes" exact component={ShowPage}></Route>
    );
    expect(componentWrapper).toMatchSnapshot();
  });
});

export {};
