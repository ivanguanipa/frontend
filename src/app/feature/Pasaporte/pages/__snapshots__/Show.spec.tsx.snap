// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Page Test shallow render 1`] = `
<Layout
  description="Ver de pasaporte"
  title="Pasaportes"
>
  <Connect(MostrarPasaportes) />
</Layout>
`;
